_data/other.yml

```diff
@@ -167,7 +167,8 @@ websites:
     tfa:
       - sms
       - totp
-    doc: https://help.figma.com/article/208--
+      - proprietary
```

Comment:
Are you sure it's not just TOTP?

Reply:
Duo provides push notifications with a simple tap verification as well, doesn't it?

Reply:
Yes but they might just mean that you can scan the RFC-6238 QR code in the Duo app and use it like Google Authenticatior.